### PR TITLE
Use hash when searching

### PR DESCRIPTION
**Pull request by rvsia**
This appends hash to algolia suggestions.

### CODE REVIEW

File: packages/react-renderer-demo/src/app/src/components/docsearch.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ const DocSearch = () => {
       handleSelected: (input, event, suggestion) => {
         event.button = 0;
         const resultUrl = new URL(suggestion.url);
-        push(resultUrl.pathname);
+        push(`${resultUrl.pathname}${resultUrl.hash}`);
```
Comment (Hyperkid123) [inline packages/react-renderer-demo/src/app/src/components/docsearch.js]: what if the hash is undefined?

Comment (rvsia) [reply packages/react-renderer-demo/src/app/src/components/docsearch.js]: Can it be?

Comment (Hyperkid123) [reply packages/react-renderer-demo/src/app/src/components/docsearch.js]: Well thet depends on the algolia crawler. We are not in control here

**Pull request merged**